packages/enterprise-search-universal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elastic/enterprise-search-universal",
-  "version": "8.14.0",
+  "version": "8.17.0",
```

Comment:
Does this mean the message will appear for 8.17? We should only show it for 8.18+ as the official EOL announcement will take place at the public release of 8.18/9.0 (end of March 2025)